RELEASENOTES/v2.0.0.rc-2.MD

```diff
@@ -0,0 +1,65 @@
+# Notary v2 Release Candidate 2 
```

Comment:
We should remove all references to v2 in the documents. Please use Notary Project instead. I will not comment on every line including the reference - we should do a bulk update.